src/hotspot/share/code/aotCodeCache.cpp

```diff
@@ -1266,7 +1291,7 @@ bool AOTCodeCache::finish_write() {
     copy_bytes((_store_buffer + entries_offset), (address)current, entries_size);
     current += entries_size;
 
-    log_stats_on_exit();
+    //log_stats_on_exit();
```

Comment:
Not sure if this matters since I assume this method is going to be deleted but why comment this out?

Reply:
you are right, this version of `finish_write()` would go away. I was just lazy to update the code to pass the parameter that `log_stats_on_exit()` needs :)